### PR TITLE
MP-7906 Database configuration for "InternetDomain"

### PR DESCRIPTION
**Pull request by msadikkoseaupa**

### CODE REVIEW

**Comment by gjorgji-grgovski-authoritypartners** [comment]
The title of the PRs should start with "MP-XXXX ........."

File: src/Nox.Types.EntityFramework/Types/InternetDomain/InternetDomainDatabaseConfigurator.cs [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        NoxSimpleTypeDefinition property,
+        Entity entity,
+        bool isKey)
+    {
+        builder
+            .Property(property.Name)
+            .IsRequired(property.IsRequired)
+            .IsUnicode(false)
+            .HasMaxLength(63)
+            .HasConversion<InternetDomainConverter>();
```
Comment (gjorgji-grgovski-authoritypartners) [inline src/Nox.Types.EntityFramework/Types/InternetDomain/InternetDomainDatabaseConfigurator.cs:44]: InternetDomainConverter should use "FromDatabase"

**Review by gjorgji-grgovski-authoritypartners — CHANGES_REQUESTED** [review]

**Review by dejobratic — CHANGES_REQUESTED** [review]
Looks good. Just the comment related to database converter.

File: src/Nox.Types.EntityFramework/Types/InternetDomain/InternetDomainDatabaseConfigurator.cs (hunk 1 above)
Comment (msadikkoseaupa) [reply src/Nox.Types.EntityFramework/Types/InternetDomain/InternetDomainDatabaseConfigurator.cs:44]: Required change was applied

**Review by dejobratic — APPROVED** [review]
Good job.

**Review by rochar — APPROVED** [review]

**Review by gjorgji-grgovski-authoritypartners — APPROVED** [review]

**Pull request merged**